### DISCUSSION TITLE
Impossible to add old issues to project

### DISCUSSION THREAD

**Discussion by teolemon** [community/community#14872] [Projects and Issues]
I can't seem to be able to add a few remaining issues from 2016 to my project (https://github.com/openfoodfacts/openfoodfacts-server/issues?q=is%3Aissue+is%3Aopen+no%3Aproject), either in bulk or one by one.
Anyone able to reproduce on their projects ?

**Comment by schalkneethling** [comment]
@teolemon have you checked whether there is an error in the browser console? I have run into an issue where the project board seems to have gotten too big and throws a 500 server error in the console. Not sure if this is related to the same problem: https://github.com/github/feedback/discussions/14798

**Reply by teolemon** [reply]
It's the same, we hit the 1200 threshold for a while, and I understood that later

**Reply by schalkneethling** [reply]
Thanks for the info. By the way, where did you get the info about the 1200 threshold?
